doc/cli/cli_show.rst

```diff
@@ -0,0 +1,130 @@
+    (0002, 0013) Implementation Version Name         SH: 'DCTOOL100'
+    (0002, 0016) Source Application Entity Title     AE: 'CLUNIE1'
+    -------------------------------------------------
+    (0008, 0005) Specific Character Set              CS: 'ISO_IR 100'
+    (0008, 0008) Image Type                          CS: ['ORIGINAL', 'PRIMARY', 'AXIAL']
+    (0008, 0012) Instance Creation Date              DA: '20040119'
+    (0008, 0013) Instance Creation Time              TM: '072731'
+    (0008, 0014) Instance Creator UID                UI: 1.3.6.1.4.1.5962.3
+    (0008, 0016) SOP Class UID                       UI: CT Image Storage
+    (0008, 0018) SOP Instance UID                    UI: 1.3.6.1.4.1.5962.1.1.1.1.1.20040119072730.12322
+    (0008, 0020) Study Date                          DA: '20040119'
+    .
+    .
+    .
+    (0043, 104b) [DAS xm pattern]                    SL: 0
+    (0043, 104c) [TGGC trigger mode]                 SS: 0
+    (0043, 104d) [Start scan to X-ray on delay]      FL: 0.0
+    (0043, 104e) [Duration of X-ray on]              FL: 10.60060977935791
+    (7fe0, 0010) Pixel Data                          OW: Array of 32768 elements
+    (fffc, fffc) Data Set Trailing Padding           OB: Array of 126 elements
+
+Note that prefixing the file specification with ``pydicom::`` will read the file
+from the *pydicom* test data files rather than from the file system.
+
+You can also show just parts of the DICOM file by specifying a data element
+using the usual pydicom keyword notation:
+
+.. code-block:: console
+
+    $ pydicom show pydicom::CT_small.dcm::PatientName
+    CompressedSamples^CT1
+
+    $ pydicom show pydicom::rtplan.dcm::FractionGroupSequence
+    [(300a, 0071) Fraction Group Number               IS: "1"
+    (300a, 0078) Number of Fractions Planned         IS: "30"
+    (300a, 0080) Number of Beams                     IS: "1"
+    (300a, 00a0) Number of Brachy Application Setups IS: "0"
+    (300c, 0004)  Referenced Beam Sequence  1 item(s) ----
+    (300a, 0082) Beam Dose Specification Point       DS: [239.531250000000, 239.531250000000, -751.87000000000]
+    (300a, 0084) Beam Dose                           DS: "1.0275401"
+    (300a, 0086) Beam Meterset                       DS: "116.0036697"
+    (300c, 0006) Referenced Beam Number              IS: "1"
+    ---------]
+
+The ``-q`` quiet argument shows a minimal version of some of the information in the
+file, using just the DICOM keyword and value (not showing the tag numbers
+and VR). The example below shows the quiet mode with an image slice::
+
+    pydicom show -q pydicom::ct_small.dcm
+
+    SOPClassUID: CT Image Storage
+    PatientName: CompressedSamples^CT1
+    PatientID: 1CT1
+    StudyID: 1CT1
+    StudyDate: 20040119
+    StudyTime: 072730
+    StudyDescription: e+1
+    Image: 16-bit CT 128x128 pixels Slice location: -77.2040634155
+
+And the following example shows an RT Plan in quiet mode::
+
+    pydicom show -q pydicom::rtplan.dcm
```

Comment:
I'll change it to a one-line per data element image info then since both you and @scaramallion commented on it.  It also makes more sense if in future adding multi-file support, then could make a sort of 'table' of the values across different files.